Replace image on video.wikia.com mainpage

macbre opened the pull request:
https://wikia-inc.atlassian.net/browse/SUS-1118

Use new logo provided by the Design Team, crop and resize it to match the existing one. And run `optipng` on the sprite generated by  `generateSprites.php` script.

New logo is not transparent, but [Wikia Video](http://video.wikia.com] has a white background.

@mixth-sense / @Grunny / @TK-999

macbre committed:
logo_wikiavideo.png: new asset for Wikia Video

macbre committed:
extensions/wikia/VideoPageTool/images/sprite-VideoPageTool.png regenerated using generateSprites.php script

macbre committed:
Run optipng on sprite-VideoPageTool.png

OptiPNG 0.6.4: Advanced PNG optimizer.
Copyright (C) 2001-2010 Cosmin Truta.

** Processing: sprite-VideoPageTool.png
839x55 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 22187 bytes
Input file size = 22289 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0		IDAT size = 19187

Selecting parameters:
  zc = 9  zm = 8  zs = 0  f = 0		IDAT size = 19187

Output IDAT size = 19187 bytes (3000 bytes decrease)
Output file size = 19265 bytes (3024 bytes = 13.57% decrease)

mszabo-wikia reviewed (approved):
LGTM
[![LGTM](https://lgtm.in/p/qPLT76z2r)](https://lgtm.in/i/qPLT76z2r)

Grunny reviewed (approved):
LGTM.

macbre committed:
Merge branch 'dev' into SUS-1118-new-wikia-video-logo

macbre committed:
VideoPageTool: transparent logo_wikiavideo.png

macbre committed:
Run optipng on images/sprite-VideoPageTool.png

OptiPNG 0.6.4: Advanced PNG optimizer.
Copyright (C) 2001-2010 Cosmin Truta.

** Processing: images/sprite-VideoPageTool.png
839x55 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 17408 bytes
Input file size = 17510 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0		IDAT size = 14307

Selecting parameters:
  zc = 9  zm = 8  zs = 0  f = 0		IDAT size = 14307

Output IDAT size = 14307 bytes (3101 bytes decrease)
Output file size = 14385 bytes (3125 bytes = 17.85% decrease)
